src/StatsdClient/Worker/AsynchronousWorker.cs

```diff
@@ -111,6 +112,15 @@ private void Dequeue()
                         }
                     }
                 }
+#if !NETSTANDARD1_3
```

Comment:
Can you give more information on why using `!NETSTANDARD1_3`?  Should it be `NETFRAMEWORK`?

Reply:
Yeah, good catch, I don't think there was a reason.  `NETFRAMEWORK` will be better for adding/changing targets, which I did use on the test. 🤷‍♂️

Reply:
Ah, so I remembered today that I was thinking since ThreadAbortException exists in .NET Standard 2.0, and a .NET Standard DLL _could_ run under the .NET Framework this would allow the fix to work in that scenario, while the extra code would be meaningless when it ran under .NET Core.  That said, since you include assemblies targeting the .NET Framework in the NuGet, I'm not sure you can get to that scenario under normal circumstances.

Reply:
Thanks for the detailed explanation!